### ISSUE TITLE
B9 fatal exception when SimpleConstruction is installed

### ISSUE THREAD

**Issue by noelforte** [CobaltWolf/Bluedog-Design-Bureau#1360]
Getting a unhandled exception from B9 on game load that I've traced back to `SimpleConstruction `being installed. The exception appears to be related to #1319 and #1327, but even with the latest version of BDB continues to be thrown.

Here's a copy of my logfile: [KSP.log](https://github.com/CobaltWolf/Bluedog-Design-Bureau/files/11023457/KSP.log)

**Comment by noelforte** [comment]
Took a crack at a potential fix that might be way off but does prevent the exception from being thrown. Submitted in pull #1361

**Comment by Rodg88** [comment]
Yeah that fix is already made on dev branch, it will be in next release
